lib/travis/hub/service/error_job.rb

```diff
@@ -0,0 +1,55 @@
+require 'travis/hub/helper/context'
+require 'travis/hub/model/job'
+require 'travis/hub/support/logs'
+
+module Travis
+  module Hub
+    module Service
+      class ErrorJob < Struct.new(:data)
+        include Helper::Context
+
+        MSGS = {
+          error:          'Erroring <Job id=%s>. Reason: %s',
+          resets_limited: 'Resets limited: %{resets}'
+        }
+
+        LOGS = {
+          resets_limited: 'Automatic restarts limited: Please try restarting this job later or contact support@travis-ci.com.',
+        }
+
+        def run
+          logger.error MSGS[:error] % [job.id, reason]
+          error
+          update_log
+        end
+
+        private
+
+          def error
+            job.finish!(state: :errored)
+          end
+
+          def update_log
+            logs.update(id, LOGS[:resets_support])
```

Comment:
I don't see `LOGS[:resets_support]` being defined anywhere, should it be `LOGS[:resets_limited]`?